portfolio/src/main/webapp/script.js

```diff
@@ -87,43 +78,32 @@ function createListElement(text) {
   return liElement;
 }
 
-/** Fetches tasks from the server and adds them to the DOM. */
-function loadTasks() {
-  fetch('/list-post').then(response => response.json()).then((tasks) => {
-    const taskListElement = document.getElementById('post-list');
-    tasks.forEach((task) => {
-      taskListElement.appendChild(createTaskElement(task));
-    })
-  });
-}
 
-/** Creates an element that represents a task, including its delete button. */
-function createTaskElement(task) {
-  const taskElement = document.createElement('li');
-  taskElement.className = 'task';
 
-  const titleElement = document.createElement('span');
-  titleElement.innerText = task.title;
+  
+
 
-  const deleteButtonElement = document.createElement('button');
-  deleteButtonElement.innerText = 'Delete';
-  deleteButtonElement.addEventListener('click', () => {
-    deleteTask(task);
 
-    // Remove the task from the DOM.
-    taskElement.remove();
-  });
 
-  taskElement.appendChild(titleElement);
-  taskElement.appendChild(deleteButtonElement);
-  return taskElement;
+function getComment() {
+    var limit = document.getElementById("num-results").value;
+  var limitParsed = +limit;
+  console.log(limitParsed);
+  fetch('/list-post').then(response => response.json()).then((comment) => {
+    const statsListElement = document.getElementById('post-list');
+     statsListElement.innerHTML = '';
+    for (i = 0; i < limitParsed; i++) {
```

Comment:
i'm surprised this for loop works, as you usually need to initialize "i" with "let i" or "var i". Is this behaving as intended?

Reply:
I accidentally did not initialize it, it was working well though. Anyways, I will modify my code to follow rules and avoid bugs.